contracts/ERC721A.sol

```diff
@@ -464,6 +466,10 @@ contract ERC721A is IERC721A {
      * - `to` cannot be the zero address.
      * - `quantity` must be greater than 0.
      *
+     * Note: some marketplaces may have difficulties fully registering a very large `quantity`.
```

Comment:
This note is not applicable for this function.

OpenSea and LooksRare support this `_mint` implementation with any quantity of tokens so long as the transaction is posted to the ledger. This is because this transaction would fail with out of gas if trying too large a quantity.

Reply:
Manual emprical testing on their Rinkeby testnet marketplaces shows that they are incappable of supporting any quantity of tokens though.

Reply:
Please post the testing tx you are talking about